docs/codebase-best-practices.md

```diff
@@ -0,0 +1,25 @@
+# Codebase Best Practices
+
+## General JavaScript
+
+## Specific TypeScript
```

Comment:
This would be a good place to include our naming conventions for interfaces, types, etc. 

Once we can land on a standard, of course. 🙂

Reply:
Exactly the point of this 😉